app/assets/stylesheets/partials/_footer.scss

```diff
@@ -1,27 +1,65 @@
 footer {
-  background-color: grey;
+  display: flex;
```

Comment:
Overall looks good! Could I suggest a layout like this for the bottom of the footer in the mobile view?

![screen shot 2018-09-13 at 3 56 03 pm](https://user-images.githubusercontent.com/6565554/45512406-a4436a80-b76d-11e8-9d3b-6dcb92364aa0.png)

Reply:
Not a bad idea but I am going to punt on this since the distance between this and the desktop version is decently large and I don't think we win a whole lot by throwing effort at it. I also imagine DeeDee will have thoughts on restyling once she sees it.